### ISSUE TITLE
Error message for invalid YAML

### ISSUE THREAD

**Issue by feutl** [cupcakearmy/autorestic#163]
**Describe the bug**
I installed `autorestic` using `wget` and the install script as mentioned in the instructions.
Created my first yml and ended up right here :D when running `autorestic` as a user, but even with sudo it does not change much.

```
herbert@cloud:~$ autorestic check
cannot find configuration file '.autorestic.yml' or '.autorestic.yaml'.
herbert@cloud:~$ autorestic check -c .autorestic.yml
cannot find configuration file '.autorestic.yml' or '.autorestic.yaml'.
herbert@cloud:~$ autorestic check -c /home/herbert/.autorestic.yml
cannot find configuration file '.autorestic.yml' or '.autorestic.yaml'.
herbert@cloud:~$ ls -la
total 88
drwxr-xr-x 7 herbert herbert  4096 Mar  8 10:02 .
drwxr-xr-x 4 root    root     4096 Apr 11  2019 ..
-rwxrwxrwx 1 herbert herbert   193 Mar  8 10:02 .autorestic.yml
-rw------- 1 herbert herbert 30521 Mar  7 21:19 .bash_history
-rw-r--r-- 1 herbert herbert   220 Apr 11  2019 .bash_logout
-rw-r--r-- 1 herbert herbert  3526 Apr 11  2019 .bashrc
drwx------ 4 herbert herbert  4096 Jul 19  2019 .config
drwx------ 3 herbert herbert  4096 Jul 19  2019 .gnupg
drwxr-xr-x 2 herbert herbert  4096 Oct 19  2020 .nano
-rw-r--r-- 1 herbert herbert   675 Apr 11  2019 .profile
drwxr-xr-x 2 herbert herbert  4096 Dec 21  2019 .ssh
herbert@cloud:~$
```
I also changed the permissions of the file to `777`, same behavior.

The file includes the following
```
# cat .autorestic.yml
locations
  files:
    from:
      - /media/files
    to: backuppi

backends:
  backuppi:
    type: rest
    path: http://x.x.x.x:8000/files
```

**Environment**
OS: Debian
Autorestic: autorestic version 1.5.5

Thanks

**Comment by cupcakearmy** [comment]
I can't reproduce it. Have you solved your issue?

**Comment by feutl** [comment]
no, and I gave up on this.

**Comment by garettmd** [comment]
I'm also facing this issue, on autorestic 1.5.7, restic 0.12.1

**Comment by hogcycle** [comment]
Same here. Even when explicitly defining the path with -c, still doesn't work.
![image](https://user-images.githubusercontent.com/57007241/158080235-38fac359-e02c-4c09-aba8-8adf6821184b.png)

**Comment by aniqueta** [comment]
I'm experiencing the same issue, setting up autorestic for the first time. It cannot find the config file, even with -c or --config specified. --verbose doesn't show anything new to help debug. Sudo, changing permissions, and moving the config elsewhere doesn't resolve this either. Appreciate the help resolving this.

**Comment by tcamargo** [comment]
Trying autorestic for the first time. Same problem.

```
$ sudo docker run --rm \
>   -v /volume1/workspace/mydotfiles/rclone/luggage-autorestic.yml/:/autorestic.yml \
>   -v /volume1/workspace/mydotfiles/rclone/rclone.conf:/root/.config/rclone/rclone.conf:ro \
>   -v /volume1/photos/originals:/volume1/photos/originals \
>   cupcakearmy/autorestic \
>   autorestic backup -va -c /autorestic.yml
cannot find configuration file '.autorestic.yml' or '.autorestic.yaml'. 
```

```
$ sudo docker run --rm -it  -v /volume1/workspace/mydotfiles/rclone/luggage-autorestic.yml/:/autorestic.yml   -v /volume1/workspace/mydotfiles/rclone/rclone.conf:/root/.config/rclone/rclone.conf:ro   -v /volume1/photos/originals:/volume1/photos/originals   cupcakearmy/autorestic /bin/bash
bash-5.1# ls
autorestic.yml  etc             media           proc            sbin            tmp             volume1
bin             home            mnt             root            srv             usr
dev             lib             opt             run             sys             var
```

**Comment by cupcakearmy** [comment]
Thaks for all your feedback. I've tried macOS and linux servers, but i really can't reproduce :/ I would love to fix that.
What plaforms are you running on? @tcamargo @aniqueta @hogcycle @garettmd

**Comment by dev-zero** [comment]
I just observed the same error message on macOS after a change to the configuration: turns out that an error in a cron line (e.g. a stray quote) made the yaml invalid and also leads to the same error message:

```yaml
    cron: ''0 0 * * *'
```

**Comment by cupcakearmy** [comment]
Oh that is so stupid. Can cofirm. So it's the same error message if yaml is not valid. That is stupid behavior on part of `autorestic`. Will fix asap

**Comment by cupcakearmy** [comment]
Should be fixed with `v1.5.8`. I'll close it for now. Feel free to reopen if it doesn't work.
